src/visitor.rs

```diff
@@ -671,6 +689,20 @@ impl<'b, 'a: 'b> FmtVisitor<'a> {
         false
     }
 
+    fn is_rustfmt_macro_error(&self, segments: &[ast::PathSegment]) -> bool {
```

Comment:
I think the following method name sounds better ? It would mean the method would do the opposite

```suggestion
    fn is_rustfmt_skip_attr(&self, segments: &[ast::PathSegment]) -> bool {
```